.gitignore

```diff
@@ -232,3 +232,6 @@ typings/
 
 # vagrant
 .vagrant/
+
+# auto-generated developmend version of settings
+_dev_settings.yml
```

Comment:
We have established convention of suffix naming.
We have `settings.yml` and `settings_test.yml`, why not `settings_dev.yml`?